docs/app/templates/getting-started/setup.hbs

```diff
@@ -49,86 +53,49 @@ See the
 >Ember documentation</a>
 for more information about including assets into your app.
 
-<h3>Using the default blueprint</h3>
-
-<p>
-  By default, ember-bootstrap installs Bootstrap 5 and uses the installed
-  preprocessor if there is one. You can use the ember-bootstrap default
-  blueprint to switch the Bootstrap version or preprocessor.
-</p>
-
-<h4>Use a CSS preprocessor</h4>
+<h3>Use a CSS preprocessor</h3>
 
 <p>To use a Sass as a CSS preprocessor instead of plain CSS (highly
-  recommended!), you can use this command to install and set this up:</p>
-
-<pre class="highlight">ember generate ember-bootstrap --preprocessor=sass</pre>
+  recommended!), you can install and setup
+  <a
+    href="https://github.com/adopted-ember-addons/ember-cli-sass"
+  >ember-cli-sass</a>-
+</p>
 
 <p>See the
   <LinkTo @route="getting-started.assets">Assets guide</LinkTo>
   for details.</p>
 
-<h4>Switch to Bootstrap 5</h4>
-
-<p>You can switch to Bootstrap 5 by running this command:</p>
-
-<pre class="highlight">ember generate ember-bootstrap --bootstrap-version=5</pre>
-
-<BsButton @type="info" @onClick={{this.toggleBlueprintDetails}}>
-  About the default blueprint
-</BsButton>
-
-<BsCollapse @collapsed={{this.blueprintDetailsCollapsed}}>
-
-  <BsAlert class="mt-4" @type="info" @dismissible={{false}}>
-
-    <h4>About the default blueprint</h4>
-
-    <p>The default blueprint does the following:</p>
-
-    <ul>
-      <li>
-        Removes any unneeded versions of bootstrap and bootstrap-sass from
-        <code>package.json</code>
-        and
-        <code>bower.json</code>
-      </li>
-      <li>
-        Installs the appropriate version of bootstrap or bootstrap-sass as an
-        npm package
-      </li>
-      <li>
-        Removes unneeded dependencies on ember-cli-less and ember-cli-sass
-      </li>
-      <li>
-        Installs
-        <code>ember-cli-sass</code>
-        and
-        <code>app.scss</code>
-        if appropriate
-      </li>
-      <li>
-        Adds the appropriate "@import" statement to your
-        <code>app.scss</code>
-        if it&apos;s not there already
-      </li>
-      <li>
-        Safely edits your
-        <code>ember-cli-build.js</code>
-        to ensure the proper ember-bootstrap settings for your configuration
-      </li>
-    </ul>
-  </BsAlert>
-</BsCollapse>
+<h3>Switch to Bootstrap 4</h3>
+
+<p>Ember Bootstrap defaults to Bootstrap 5 but supports Bootstrap 4 in parallel.
+  You can switch to Bootstrap 4 by setting the
+  <code>bootstrapVersion</code>
+  option to
+  <code>4</code>
+  in your
+  <code>ember-cli-build.js</code>:
+
+  {{! prettier-ignore }}
+  <pre class="highlight"><code
+>module.exports = function (defaults) {
+  const app = new EmberApp(defaults, {
+    'ember-bootstrap': {
```

Comment:
I wonder if these fields should be changed to `@embroider-macros`' `setConfig` field.

I'm not well versed yet in how Vite apps / future EmberJS apps should behave. Is this change something we can attempt through the ember-try scenarios?